CONTRIBUTING.md

```diff
@@ -59,6 +59,12 @@ Changes should conform to:
 
 With the exception that 4-space indendation be used.
 
+#### General guidelines for Swift for TensorFlow API docs contribution
+
+* Closely mirror Swift for TensorFlow API docs with implementations in [tensorflow/tensorflow/python](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/python) to maintain consistency across platforms.
+  - If required, replace variable names from Python docs to reflect the Swift code. For example: `y_true` and `y_predict` in TensorFlow 2.x API docs will become `expected` and `predicted` in Swift for TensorFlow, respectively.
+* When you contribute a new feature to Swift for TensorFlow, the maintenance burden is (by default) transferred to the Swift to TensorFlow team. This means that the benefit of the contribution must be compared against the cost of maintaining the feature.
```

Comment:
Thanks for pulling this over! While I think this is true, I think I would like to strike a different tone. What do you (and everyone else too!) think about the following alternative:

```suggestion
* `swift-apis` is a carefully curated set of maintained APIs and functionality. We generally aim to incubate new features in the library ecosystem that builds on top of S4TF. Popular functionality (such as new layers or helpful abstractions) often graduate into `swift-apis` after maturing in other repositories (e.g. `swift-models`) first, where we can try them out in context and iterate quickly.
```

Reply:
I think moving this bullet to a separate section (not under API documentation guidelines) makes sense!

Reply:
Awesome. I took @saeta 's new paragraph, changed it just a little:
> "...that builds on top of **Swift for TensorFlow** ~~S4TF~~... 
>
> Popular functionality,~(~ such as new layers or helpful abstractions,  ~)~..."
> 
> "...after maturing in other repositories **first** (e.g. `swift-models`) ~~first~~, where **the development team and the community** ~~we~~ can try them out in context and iterate quickly..."

and, as per @dan-zheng 's proposal, moved it outside. Perhaps, a good place for it be could be in the beginning, since the new stuff summarizes `swift-apis` nicely. And, to make our community members feel even more welcome to contribute from the very beginning, I called that section "Welcome!", although it may sound cheesy but I've run out of ideas 🧀 

LMKWYT

![](http://i0.wp.com/wahlnetwork.com/wp-content/uploads/2016/03/Tron-fight-for-the-user.jpg?fit=873%2C428)